src/main/java/com/impactupgrade/integration/hubspot/model/ContactProperties.java

```diff
@@ -1,21 +1,43 @@
 package com.impactupgrade.integration.hubspot.model;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public class ContactProperties implements AbstractModel {
 
   private HasValue<String> firstname;
   private HasValue<String> lastname;
+  @JsonProperty("email") 
   private HasValue<String> email;
+  @JsonProperty("phone") 
   private HasValue<String> phone;
+  @JsonProperty("home_phone")
+  private HasValue<String> homePhone;
+  @JsonProperty("mobile_phone") 
+  private HasValue<String> mobilePhone;
+  @JsonProperty("work_phone") 
+  private HasValue<String> workPhone;
+  @JsonProperty("preferred_phone") 
+  private HasValue<String> preferredPhone;
+  @JsonProperty("address") 
+  private HasValue<String> address;
+  @JsonProperty("city") 
+  private HasValue<String> city;
+  @JsonProperty("state") 
+  private HasValue<String> state;
+  @JsonProperty("zip") 
+  private HasValue<String> zip;
+  @JsonProperty("country") 
+  private HasValue<String> country;
 
-  public HasValue<String> getFirstname() {
+  public HasValue<String> getFirstName() {
```

Comment:
Need to change this back to getFirstname -- for POJO rules, reflection, Jackson, etc., the getters and setters must match the case of the field name (with the exception of the first letter capitalized)